src/terraform/command_helpers.go

```diff
@@ -148,3 +132,12 @@ func getDependencyServiceEnvVars(deployConfig deploy.Config, serviceConfig types
 	}
 	return result
 }
+func extractDependencyData(dependencyName string, deployConfig deploy.Config) (string, error) {
+	serviceData := deployConfig.AppContext.GetDependencyServiceData(dependencyName)
+	serviceDataBytes, err := json.Marshal(serviceData)
+	if err != nil {
+		return "", err
+	}
+	keyName := strings.ToUpper("SERVICE_DATA")
```

Comment:
I think ToUpper / the variable keyName is not unneeded. Can we update the unit tests to validate that the key is `SERVICE_DATA`. I would have liked a test failure when we had used the wrong key name